### PR TITLE
doc: add TLM documentation

### PR DESCRIPTION
**Pull request by inkhey**
related to #4578 

:warning: Do not forget to deprecate our internal doc as soon as this document is merged: https://algoo.tracim.fr/ui/workspaces/49/contents/html-document/12987 to not have to maintain 2 different documents.

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Comment by grignards** [comment]
@inkhey Would it be difficult to integrate this documentation directly in the generated API documentation available in `/api/doc`? This would help developers find it I think.

File: backend/doc/live_messages.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+```
+
+## TLM Types
+
+Possible core event types:
+
+    created (C below)
+    modified (M below)
+    deleted (D below)
+    undeleted (U below)
```
Comment (raphj) [inline backend/doc/live_messages.md:39]:
```suggestion
- created (C below)
- modified (M below)
- deleted (D below)
- undeleted (U below)
```

File: backend/tracim_backend/lib/core/event.py [hunk 2]
```diff
@@ -877,6 +877,12 @@ def get_event_user_id(session: TracimSession, event: Event) -> typing.Optional[i
 
 
 def _get_user_event_receiver_ids(event: Event, session: TracimSession, config: CFG) -> Set[int]:
+    """
+    User event are received by :
+    - user itself
+    - administrators
+    - user that known the user (in same workspace)
```
Comment (raphj) [inline backend/tracim_backend/lib/core/event.py:884]:
```suggestion
    User event are received by :
    - the user themself
    - administrators
    - users knowing the user (in same workspace)
```

**Comment by inkhey** [comment]
@grignards I added a link in the api doc to make usage of TLM a bit smoother, for now the link is broken until we merge this pr into develop.

File: backend/doc/live_messages.md [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+| user                   | C/M/D/U          |               | author,user                            |   author can be null (tracimcli),   user account modification, user creation/disabling/etc  | administrators, user itself, user in at least one same space                        |
+| workspace              | C/M/D/U          |               | author,workspace                       | Space creation/deletion but also description/name update                                    | same as workspace_members if confidential, all users if not (open/on_request space) |
+| workspace_member       | C/M/D            |               | author,user,workspace,member           |  add/remove members in space but also role change                                            | administrators, user themself (if one), space members                                 |
+| content                | C/M/D/U          | any content type, e.g "thread"  | author,workspace,content               | any content modification(create/update/deletion,etc) | space members                                                                       |
```
Comment (gbonaspetti) [inline backend/doc/live_messages.md:64]:
```suggestion
| content                | C/M/D/U          | any content type, e.g. "thread"  | author,workspace,content               | any content modification(create/update/deletion,etc) | space members                                                                       |
```

File: backend/doc/live_messages.md [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+| reaction               | C/D              |               | author,workspace,content,reaction,user | emoji reaction on content/comment                                                           | space members                                                                       |
+| workspace_subscription | C/M/D            |               | author, workspace, subscription, user  | subscription for on request space (with validation mecanism)                                | administrator, subscription author, workspace managers                              |
+| tag                    | C/M/D            |               | author,workspace,tag                   | tag usable in space                                                                         | same as workspace                                                                   |
+| content_tag            | C/D              |               | author,workspace,tag,content           | Tag associated to a content                                                                  | space members                                                                       |
```
Comment (gbonaspetti) [inline backend/doc/live_messages.md:69]:
```suggestion
| content_tag            | C/D              |               | author,workspace,tag,content           | tag associated to a content                                                                  | space members                                                                       |
```

**Review by gbonaspetti — APPROVED** [review]

**Review by Haskaris — APPROVED** [review]

**Pull request merged**